data/gegno/gegno events.txt

```diff
@@ -29,6 +29,28 @@ event "gegno: quarg talks"
 event "gegno: apprehension"
 
 
+event "gegno: vi occupy zydee"
+	system "Vesvii"
+		fleet "Scin Warfleet" 2700
+		fleet "Vi Warfleet" 2400
+	system "Kneirc"
+		government "Gegno Vi"
+		attributes "gegno" "vi"
+		fleet "Small Vi" 1800
+		fleet "Vi Warfleet" 4600
+		fleet "Small Scin" 2900
+		fleet "Scin Warfleet" 4000
+	planet "Zydee"
+		attributes "gegno" "vi" "moon"
+		description `There is evidence that the nearby Gegno Scin have attempted to establish a presence on this barren moon in the form of several partially-constructed buildings and landing zones scattered about the surface. In recent times, however, the opposing Gegno Vi have fielded their own forces to take control of the system, and have begun a number of operations to convert existing infrastructure and set up posts for their own, claiming the moon in their name.`
+		spaceport `The Vi have made makeshift landing zones and outposts, such as this one, for the group of warriors they've distributed across the moon. Although their numbers are no more than than that of a small village, those who are present are hard-working and surprisingly have accomplished a lot during their time on the surface while the fleets above defend them.`
```

Comment:
Personally I think that having both `hard-working and surprisingly have accomplished a lot` is a bit redundant. Up to you though.
```suggestion
		spaceport `The Vi have made makeshift landing zones and outposts, such as this one, for the group of warriors they've distributed across the moon. Although their numbers are no more than than that of a small village, those who are present have accomplished a considerable amount during their time on the surface while the fleets above defend them.`
```